src/citrine/informatics/workflows/design_workflow.py

```diff
@@ -24,8 +24,6 @@ class DesignWorkflow(Resource['DesignWorkflow'], Workflow, AIResourceMetadata):
         if none is provided, one matching your design space will be automatically generated
     predictor_id: UUID
         the UUID corresponding to the predictor to use
-    project_id: UUID
```

Comment:
Nice! If you want to go above and beyond the ticket, the docstring is also missing the `description` param

Reply:
Done! I added it below. What do you think?